clang/include/clang/CIR/Interfaces/CMakeLists.txt

```diff
@@ -22,3 +22,4 @@ endfunction()
 
 add_clang_mlir_attr_interface(ASTAttrInterfaces)
 add_clang_mlir_op_interface(CIROpInterfaces)
+add_clang_mlir_op_interface(LoopOpInterface)
```

Comment:
Can you please name this `CIRLoopOpInterfaces`? `ASTAttrInterfaces` got there before but I haven't had the chance to change it. Another option is to change the cmake function and just make sure all exported content from said interfaces are prefixed with CIR. I don't really mind if the header file and classes have such prefix.

Reply:
I'm also not seeing stuff like `MLIRCIRASTAttrInterfacesIncGen` getting added to other cmake files. Maybe you forgot those?